build-system/tasks/bundle-size/APPROVERS.json

```diff
@@ -3,20 +3,40 @@
     "approvers": ["ampproject/wg-performance"],
     "threshold": 0.1
   },
+  "dist/v0.mjs": {
```

Comment:
@danielrozenberg How hard would it be to allow using globs/regex for these? It seems like there will rarely be a case where the approvers should be any different between the two

Reply:
Probably easy, dunno if it's worth bothering though given that nobody edits this file 🙃